Dockerfile.dashboard

```diff
@@ -2,12 +2,12 @@ FROM python:2.7
 
 WORKDIR /code/dashboard
 
-ADD dashboard/requirements_prod.txt /code/dashboard/requirements_prod.txt
-RUN pip install -r requirements_prod.txt
-ADD dashboard/requirements.txt /code/dashboard/requirements.txt
+ADD dashboard/requirements.txt .
 RUN pip install -r requirements.txt
 
-ADD models.py /code/models.py
-ADD settings.py /code/settings.py
-ADD dashboard /code/dashboard
-CMD ["python", "server.py"]
+ADD dashboard .
+ADD models.py .
+ADD settings.py .
+
+ENTRYPOINT ["python", "server.py"]
```

Comment:
Why `ENTRYPOINT` instead of `CMD`?
An opinion from internet:
>Given how much easier it is to override the CMD, the recommendation is use CMD in your Dockerfile when you want the user of your image to have the flexibility to run whichever executable they choose when starting the container.

Currently, we need to run migrations, test and shell on the same image, so this change doesn't help with that :(

Reply:
Not exactly. In this case if the main purpose of this image is to run service, and there are other exectuables which are run not very often it's best to define ENTRYPOINT for that service, and CMD - default parameters for that service

Reply:
Best practice is to have 1 application in container